src/sentry/shared_integrations/response/base.py

```diff
@@ -59,6 +59,8 @@ def from_response(
         elif response.text.startswith("<"):
             if not allow_text:
                 raise ValueError(f"Not a valid response type: {response.text[:128]}")
+            elif response.status_code == 301 and ignore_webhook_errors:
+                return
             elif response.status_code < 200 or response.status_code >= 300:
```

Comment:
Do we still want to raise on other 30x codes? If not we could just raise the elif to `>= 400` right?

Reply:
I like that idea - I was going pretty specifically off of the events in https://sentry.io/organizations/sentry/issues/2929617510/?project=1&referrer=slack (all 301s) but you're right, we don't care about 3XX